docs/application/dotnet/guides/nui/xaml/define-layout.md

```diff
@@ -0,0 +1,151 @@
+
+  <View Name="num_area" Size2D="440,160" Position2D="1431,51">
+    <ImageView Name="photo01" ResourceUrl="*Resource*/num_w_2.png" Position2D="0,0" Size2D="110,160" />
+    <ImageView Name="photo02" ResourceUrl="*Resource*/num_w_4.png" Position2D="110,0" Size2D="110,160" />
+    <ImageView Name="photo03" ResourceUrl="*Resource*/num_w_th.png" Position2D="220,0" Size2D="220,160" />
+  </View>
+
+  <View Name="mon_area" Size2D="440,194" Position2D="1431,211">
+    <ImageView Name="photo10" ResourceUrl="*Resource*/mon_w_12.png" Position2D="0,0" Size2D="440,194" />
+  </View>
+
+  <View Name="year_area" Size2D="440,210" Position2D="1431,425">
+    <ImageView Name="photo21" ResourceUrl="*Resource*/year_w_2.png" Position2D="0,0" Size2D="110,210" />
+    <ImageView Name="photo22" ResourceUrl="*Resource*/year_w_0.png" Position2D="110,0" Size2D="110,210" />
+    <ImageView Name="photo23" ResourceUrl="*Resource*/year_w_1.png" Position2D="220,0" Size2D="110,210" />
+    <ImageView Name="photo24" ResourceUrl="*Resource*/year_w_9.png" Position2D="330,0" Size2D="110,210" />
+  </View>
+
+</ContentPage>
+```
+
+> **Note**
+>
+> In this code, the relative path of the image (relative to the location of the exe file) is used. You can also use the absolute path of the image.
+
+```xml
+<ContentPage x:Class="Tizen.NUI.Examples.AmbientPage"
+    xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+    ...
+    ...
+</ContentPage>
+```
+
+```csharp
+namespace Tizen.NUI.Examples
+{
+    public class AmbientPage : ContentPage
+    {
+        public AmbientPage(Window win) : base (win)
+        {
+            InitializeComponent();
+        }
+        ...
+    }
+}
+```
+For more information, see [XAML Support for Tizen.NUI](./getting-started-with-nui-xaml.md).
+
+## Load XAML
+
+After creating the layout, you must load the XAML file in your app code.
+First, you need to define a class named `AmbientPage` in the `Tizen.NUI.Examples` namespace as shown in the following code:
+
+```csharp
+namespace Tizen.NUI.Examples
+{
+    public class AmbientPage : ContentPage
+    {
+        public AmbientPage(Window win) : base (win)
+        {
+            InitializeComponent();
+        }
+
+        protected override void Dispose(DisposeTypes type)
+        {
+            if (disposed)
+            {
+                return;
+            }
+            if (type == DisposeTypes.Explicit)
+            {
+            }
+            base.Dispose(type);
+        }
+    }
+}
+```
+
+`AmbientPage` must inherit form `ContentPage`. All the objects that you have defined in the XAML file must be added to `ContentPage`, and then these objects will be added to `Window` as a result. As you can see the root node of the XAML file is also a `ContentPage`, which is consistent with the code. All the pages in your application must be inherited from `ContentPage`.
```

Comment:
Please use article "a" before "`ContentPage`, which is consistent" as `ContentPage` is a noun here.

```suggestion
`AmbientPage` must inherit form `ContentPage`. All the objects that you have defined in the XAML file must be added to `ContentPage`, and then these objects will be added to `Window` as a result. As you can see the root node of the XAML file is also `ContentPage`, which is consistent with the code. All the pages in your application must be inherited from `ContentPage`.
```